repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -83,7 +83,14 @@ export interface Errors {
 }
 
 export interface TicketRepresentationAttribute {
-    name: 'geoZone' | 'multipleServices' | 'hybrid' | 'pointToPointPeriod' | 'multipleServicesPricedByDistance';
+    name:
+        | 'geoZone'
+        | 'multipleServices'
+        | 'hybrid'
+        | 'pointToPointPeriod'
+        | 'multipleServicesPricedByDistance'
+        | 'multipleServicesFlatFareMultiOperator'
```

Comment:
Have we changed the code which stores the ticket representation as these new values?

Reply:
Was it in a previous PR but this interface wasn't updated?

Reply:
this was added with new values